anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -56,15 +56,11 @@ let getAnonParams (requestAnonParams: RequestAnonParams) (salt: string) =
     LayerNoiseSD = requestAnonParams.LayerNoiseSD
   }
 
-let prependStarBucketRow suppressedAnonCount result =
+let prependStarBucketRow suppressedAnonCount countColumnIndex result =
   if suppressedAnonCount <> Null then
     let starBucketRow =
       result.Columns
-      |> List.map (
-        function
-        | { Name = "count" } -> suppressedAnonCount
-        | _ -> String "*"
-      )
+      |> List.mapi (fun columnIdx _ -> if columnIdx = countColumnIndex then suppressedAnonCount else String "*")
```

Comment:
Inconsistent abbreviation of index in `columnIdx` compared to the rest of the module.